core-api/src/worker.rs

```diff
@@ -73,6 +73,13 @@ pub struct WorkerConfig {
     /// winning.
     #[builder(setter(strip_option), default)]
     pub max_task_queue_activities_per_second: Option<f64>,
+
+    /// Limits the number of activities per second that this worker will process. The worker will
+    /// not poll for new activities if by doing so it might receive and execute an activity which
+    /// would cause it to exceed this limit. Negative, zero, or NaN values will be ignored, and is
+    /// equivalent to not setting any value (ie: no rate limit).
+    #[builder(setter(strip_option), default)]
+    pub max_worker_activities_per_second: Option<f64>,
```

Comment:
Are we ignoring for the sake of langs that don't have a built-in option type? AFAIC, we can error in this case and let each lang wrap appropriately.
I'd rather that then silently ignoring invalid values.

Reply:
Option had nothing to do with it. Sure, we can error on bad values instead.